doc/build-helpers/images/dockertools.section.md

```diff
@@ -1346,6 +1350,10 @@ dockerTools.buildLayeredImage {
 []{#ssec-pkgs-dockerTools-buildNixShellImage-arguments}
 ## buildNixShellImage {#ssec-pkgs-dockerTools-buildNixShellImage}
 
+::: {.warning}
+Deprecated in favor of `streamNixShellImage` + `writeImageStream`.
```

Comment:
Please link to the docs here and in the other warning, otherwise it will be cumbersome to find what this refers to.